### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by ishpaul777**
I added linters and make simple hello microverse page with some styles.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +0,0 @@
-# Hello-Microverse
-Just the first projet in Microverse to make sure one knows basics of github-flow.
```
Comment (zhadier) [inline README.md:2]: - [ ] Kindly make sure that you add and personalize your ReadMe from the Microverse ReadMe template [here](https://github.com/microverseinc/readme-template). Make sure you keep all the main headers from the template.

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @ishpaul777  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✔️ No linter errors
✔️ GitHub flow was set up correctly
✔️ Professional pull request title and description


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishpaul777 — COMMENTED** [review]
I have added a profesional README.

File: README.md (hunk 1 above)
Comment (ishpaul777) [reply README.md:2]: Hii @zhadier how should I make changes

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @ishpaul777 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Implemented changes requested by the previous reviewer 🏄 
- Good looking README 🏂 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**